modules/ppcp-blocks/resources/js/checkout-block.js

```diff
@@ -152,6 +153,7 @@ const PayPalComponent = ({
             if (config.finalReviewEnabled) {
                 location.href = getCheckoutRedirectUrl();
             } else {
+                setGotoContinuation(true);
```

Comment:
Maybe a better name could be something like `gotoContinuationOnError` or `submittingWithoutReview`.
Right now it looks a bit confusing because normally submitting in the no-review mode should not result in continuation.